at_client/lib/src/client/at_client_impl.dart

```diff
@@ -749,8 +751,7 @@ class AtClientImpl implements AtClient {
           builder.value = await atKeyEncryption.encrypt(
               notificationParams.atKey, notificationParams.value!);
         } on KeyNotFoundException catch (e) {
-          var errorCode = AtClientExceptionUtil.getErrorCode(e);
-          return Future.error(AtClientException(errorCode, e.message));
+          return Future.error(AtClientException('AT0014', e.message));
```

Comment:
can we use error_codes['AtClientException'] instead of hard coding

Reply:
Removed the hard coding's and replaced with error_codes['AtClientException'].